less/hotgraphic.less

```diff
@@ -14,6 +14,8 @@
 		background-color: @item-text-color;
 		border-radius: 50%;
 		text-decoration: none;
+		height: (@icon-size*2);
+		width: (@icon-size*2);
```

Comment:
this change screws up the layout for `_graphicsAsPins` mode - this is what it should look like:
![hg_graphics-as-pins_correct](https://user-images.githubusercontent.com/1249597/37675285-cdb79e90-2c6c-11e8-9da8-df053d107820.png)

but this change shrinks the graphics down like this:
![hg_graphics-as-pins_broken](https://user-images.githubusercontent.com/1249597/37675307-de223d26-2c6c-11e8-8d87-796f43af741c.png)

If it helps, when in `_graphicsAsPins` mode the `<button>` elements are wrapped in a `<div>` that has a class of `.hotgraphic-narrative` (why it's called that I have NO idea!)

Reply:
Caused the `a.a11y-focusguard` element that is appended to the button once it is clicked. Fixing this with the current class and html structure seems to be impossible, as the dimension should only be set when the hotgraphic is not in "narrative" mode. We will need a class that is added to the component only when the default mode is enabled to be able to set the styling for the buttons.